h/accounts/views.py

```diff
@@ -130,8 +135,60 @@ class AsyncAuthController(AuthController):
 @view_config(attr='forgot_password', route_name='forgot_password')
 @view_config(attr='reset_password', route_name='reset_password')
 class ForgotPasswordController(horus.views.ForgotPasswordController):
-    pass
+    def forgot_password(self):
+        req = self.request
+        schema = req.registry.getUtility(IForgotPasswordSchema)
+        schema = schema().bind(request=req)
+
+        form = req.registry.getUtility(IForgotPasswordForm)
+        form = form(schema)
+
+        if req.method == 'GET':
+            if req.user:
+                return HTTPFound(location=self.forgot_password_redirect_view)
```

Comment:
Travis informs us, via `make lint`, that this is not imported. Maybe change it to `httpexceptions.HTTPFound`.